client/backend.js

```diff
@@ -162,13 +162,13 @@ module.exports = {
         });
     },
 
-    addBank: function(uuid, id, pwd, maybeWebsite) {
+    addBank: function(uuid, id, pwd, maybeCustomFields) {
         return new Promise((accept, reject) => {
             $.post('accesses/', {
                 bank: uuid,
                 login: id,
                 password: pwd,
-                website: maybeWebsite
+                customFields: maybeCustomFields ? JSON.stringify(maybeCustomFields) : maybeCustomFields
```

Comment:
I think `customFields: customFields` should work as well (`$.post` is supposed to apply `JSON.stringify` on the object)